Extension/test/scenarios/SingleRootProject/tests/lmTool.test.ts

```diff
@@ -154,7 +154,8 @@ describe('CppConfigurationLanguageModelTool Tests', () => {
                 standardVersion: 'c++20',
                 compiler: 'msvc',
                 targetPlatform: 'windows',
-                targetArchitecture: 'x64'
+                targetArchitecture: 'x64',
+                usedTestFrameworks: ['gtest', 'catch2']
```

Comment:
Should these actually match the casing of what we send? e.g. "GTest", "Catch2"?

Reply:
They do not need to be, since they are arbitrary strings used in the tests. I do agree that they should have sensible names, hence fixed as suggested.